src/Persistence/Sql.php

```diff
@@ -364,12 +364,23 @@ protected function setLimitOrder(Model $model, Query $query)
     }
 
     /**
-     * Will apply a condition defined inside $condition or $model->scope() onto $query.
+     * Will apply $model->scope() conditions onto $query.
      */
-    public function initQueryConditions(Model $model, Query $query, Model\Scope\AbstractScope $condition = null): void
+    public function initQueryConditions(Model $model, Query $query): void
```

Comment:
Probably should be `_initQueryConditions` prefixed with underscore

Reply:
why do you think so?

Reply:
Oh forget about this. I didn't notice other method below.